src/package_io/parse_input.star

```diff
@@ -401,7 +407,8 @@ def get_default_mev_params():
     return {
         "mev_relay_image": MEV_BOOST_RELAY_DEFAULT_IMAGE,
         # TODO replace with flashbots/builder when they publish an arm64 image as mentioned in flashbots/builder#105
-        "mev_builder_image": "ethpandaops/flashbots-builder:main",
+        "mev_builder_image": "flashbots/builder:1.13.2.4844.dev5-6e6c723",
```

Comment:
still nervous about having dev images as defaults, do the current ones absolutely not work? will Nethermind be alright with this @kamilchodola @piwonskp

Reply:
reverted it too.